src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -179,6 +179,10 @@ private ConfigProperties() {
     public static final String DB_URL = JPA_CONFIG_PREFIX + "hibernate.connection.url";
     public static final String DB_USERNAME = JPA_CONFIG_PREFIX + "hibernate.connection.username";
     public static final String DB_PASSWORD = JPA_CONFIG_PREFIX + "hibernate.connection.password";
+    public static final String DB_MAX_CONNECTION_ATTEMPTS = JPA_CONFIG_PREFIX +
+        "hibernate.connection.max_connection_attempts";
+    public static final String DB_CONNECTION_RETRY_INTERVAL = JPA_CONFIG_PREFIX +
+        "hibernate.connection.retry_interval";
```

Comment:
I think we can use a more generic namespace for these properties, such as `candlepin.db.*`, since there's nothing specific about jpa or hibernate to them. They're only used by candlepin logic (not passed to jpa/jdbc/hibernate) to retry the connection to the database, regardless of the underlying tech (so that their names would still make sense even if we changed from jpa/hibernate to something else)

Reply:
I agree, and that makes sense! Updated to use `candlepin.db.*`